### ISSUE TITLE
:(

### ISSUE THREAD

**Issue by RobinRMC** [Grummus/BetterLoadingScreen#9]
I loved this. Now that VRChat has implemented Easy Anti-Cheat, I'll miss this mod so much. :(

**Comment by osiris-plus** [comment]
Yeah, new loading screen just sucks. And the new music is annoying af.
